OOP/RationalNumber.cs

```diff
@@ -0,0 +1,151 @@
+﻿
+namespace OOP
+{
+    public struct RationalNumber
+    {
+        /// <summary>
+        /// Поля
+        /// </summary>
+        private int _numerator;
+        private int _denominator;
+        private float Rational=>((float)_numerator)/((float)_denominator);
```

Comment:
```suggestion
        private int _denominator;
        private double Rational=>(double)_numerator/_denominator;
```
1. Куча лишних круглых скобок, осложняющих чтение.
2. `float` имеет слишком низкую точность. Имеет смысл тогда, когда Вы экономите память, или быстродействие.

Reply:
И добавляйте пустые строки между определениями членов класса.